pkg/parse/parse.go

```diff
@@ -346,7 +345,7 @@ func getMounts(mounts []string, contextDir string) (map[string]specs.Mount, erro
 		tokens := strings.Split(arr[1], ",")
 		switch kv[1] {
 		case TypeBind:
-			mount, err := GetBindMount(tokens, contextDir)
+			mount, _, err := internalParse.GetBindMount(tokens, false, contextDir, store, "", nil)
```

Comment:
In cases where we know we mounted an image here, we definitely want to be passing the image's name (or better, its ID) back to our caller so that it can clean up after itself.